tests/eval_tests/tests.py

```diff
@@ -3,10 +3,11 @@
 import logging
 import time
 import utils
-from typing import Dict, List, Any, Optional
+from typing import Dict, List, Any, Optional, Union
 from dotenv import load_dotenv
 from llama_stack_client import LlamaStackClient
 from llama_stack_client.lib.agents.agent import Agent
+import tools as tool
```

Comment:
I see how its slightly more readable, this way. But I don't think is necessary. I would leave it as `import tools`